src/frontends/onnx/frontend/src/editor.cpp

```diff
@@ -604,5 +616,9 @@ std::vector<std::string> onnx_editor::ONNXModelEditor::get_output_ports(const Ed
 }
 
 std::shared_ptr<Model> onnx_editor::ONNXModelEditor::decode() {
-    return ngraph::onnx_import::detail::decode_to_framework_nodes(m_pimpl->m_model_proto, m_model_path, m_telemetry);
+    // TODO: REPORT PROGRESS HERE?
```

Comment:
Why not? :)

Reply:
Do we you to cover in the next PR? Do we have a jira ticket?

Reply:
looks like we started to report progress here
I guess it's better to delete TODO

Reply:
sorry, I looked at the old version, resolved